misc/vm_configs/scenarios/hybrid/nuc7i7dnb/misc_cfg.h

```diff
@@ -25,4 +25,9 @@
 				"hvlog=2M@0xe00000 "	\
 				"memmap=0x200000$0xe00000"
 
+#ifdef CONFIG_RDT_ENABLED
+#define MAX_MBA_CLOS_NUM_ENTRIES	0U
+#endif
+#define MAX_CACHE_CLOS_NUM_ENTRIES	0U
```

Comment:
The MACRO MAX_CACHE_CLOS_NUM_ENTRIES should be in #ifdef CONFIG_RDT_ENABLED.
This change need put line of "uint16_t valid_clos_num = MAX_CACHE_CLOS_NUM_ENTRIES;" and "check_vm_clos_config()" in #ifdef also.

Reply:
Agree. Will do.